bin/plugin/commands/common.js

```diff
@@ -68,5 +81,53 @@ exports.getModuleData = async ( modulesDir ) => {
 
 			return moduleData;
 		} )
-		.filter( ( moduleData ) => moduleData.name && moduleData.description );
+		.filter( ( moduleData ) => {
+			const requiredProperties = [];
+			if ( ! moduleData.name ) {
+				requiredProperties.push( 'name' );
+			}
+
+			if ( ! moduleData.description ) {
+				requiredProperties.push( 'description' );
+			}
+
+			if ( typeof moduleData.experimental === 'undefined' ) {
+				requiredProperties.push( 'experimental' );
+			}
+
+			if ( requiredProperties.length >= 1 ) {
+				const properties = requiredProperties
+					.map( ( property ) => `'${ property }'` )
+					.join( ', ' );
+				log(
+					formats.warning(
+						`This module was not included because it misses required properties: ${ properties }. \nDetails: ${ JSON.stringify(
+							moduleData
+						) }`
```

Comment:
Let's probably output formatted details:
```suggestion
						`This module was not included because it misses required properties: ${ properties }.\nDetails: ${ JSON.stringify( moduleData, null, 4 ) }`
```